seed/src/lib.rs

```diff
@@ -206,21 +206,60 @@ impl Default for NodeConfig {
     }
 }
 
+pub struct NodeHandle {
+    channel: chan::UnboundedSender<Request>,
+}
+
+impl NodeHandle {
+    pub async fn get_projects(&mut self) -> Result<Vec<()>, chan::SendError> {
```

Comment:
The one thing I'd change about the proposed API is to not leak this `SendError` as it defeats the purpose of hiding channel semantics from consumers.

Reply:
That's a :ok_hand: point

Reply:
03fa6bd